src/TokenManagers/XboxTokenManager.js

```diff
@@ -35,8 +34,9 @@ class XboxTokenManager {
     return { valid, token: token.Token, data: token }
   }
 
-  async getCachedXstsToken () {
-    const { xstsToken: token } = await this.cache.getCached()
+  async getCachedXstsToken (relyingParty) {
+    const key = createHash(relyingParty)
+    const { [key]: token } = await this.cache.getCached()
```

Comment:
OK, I see you changed the caching schema. This seems like a breaking change on behalf of the caching API, which will invalidate the current caches at minimum. Will this crash existing cached token retrieval prior to this change?

Reply:
From testing nothing was breaking, on first use after updating a new xsts token will be generated and the current xsts token keyed xststToken will stay in the cache unless manually removed. That's what I observed with the default caching and afaik this should be the same behaviour If someone has implemented their own cache using the exposed API.